src/containerbuddy/config.go

```diff
@@ -95,6 +102,10 @@ func loadConfig() (*Config, error) {
 		return nil, errors.New("More than one discovery backend defined")
 	}
 
+	if config.StopTimeout == 0 {
+		config.StopTimeout = 10
```

Comment:
The README says we default to 5. Also, [`terminate`](https://github.com/joyent/containerbuddy/pull/22/files#diff-536bffba9bba22f8afea4ecb1036fe5cR30) checks for `> 0`. It looks like we're trying to catch the struct's default value here so that we default to 5. I think this makes it impossible to intentionally set the value to 0. Maybe that's ok?